### PR TITLE
Remove most unused JavaScript, modernized it and add Alpinejs

### PR DESCRIPTION
**Pull request by imjuniper**
## What this PR does

- Removes a lot jQuery usage.
- Uses modern JavaScript features.
- Adds [Alpine.js](https://alpinejs.dev/).

## Why

- jQuery is becoming more and more unnecessary. [See this website](https://youmightnotneedjquery.com/).
- Newer JS features have good support, are easier to use and less hacky than some stuff we have now.
- Alpine.js is useful for reactivity in pages, removing the use from `querySelector` in a lot of places, making the JS more robust.

### CODE REVIEW

**Comment by jaapmarcus** [comment]
Please resolve the merge conflicts and sync up with main

File: web/templates/pages/list_weblog.php [hunk 1]
```diff
@@ -105,10 +79,14 @@
 		</div>
 	</header>
 
-	<button type="button" class="button button-secondary button-circle button-floating button-floating-top js-to-top" title="<?=_('Top');?>">
+	<a
+		href="#top"
```
Comment (AlecRust) [inline web/templates/pages/list_weblog.php:83]: Does this cause `#` in address bar and immediate jump to top on click? I just mention as previously there was no `#` and an animated slide to top.

Comment (imjuniper) [reply web/templates/pages/list_weblog.php:83]: I noticed the `#` too, but I don't know why it appears. As for the animated slide, we can easily fix that by adding [`scroll-behavior: smooth;`](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior) to the `html` selector in CSS.

Comment (AlecRust) [reply web/templates/pages/list_weblog.php:83]: Unless we intercept it with JS, the click will navigate to `#` and append it to URL in address bar. Not a big deal I guess.

I guess we should respect the user's animation preferences with regard to `scroll-behaviour`. Feels out of scope for this PR anyway 👍

**Comment by imjuniper** [comment]
Non-exhaustive summary of what I changed:

- Replaced `jQuery.cookie.js` with my own `Cookies` object.
- Removed `floating_layer.js`.
- Removed `jQuery.finder.js`.
- Added Alpine.js
  - Some page-specific code has been replaced by it (e.g. `toggle_letsencrypt`).
  - `elementHideShow` has been replaced by it. 
  - Refactors the keyboard shortcuts.
  - Notification code is all in Alpine.data.
  - `VE.tmp.form_changed` is now in Alpine.store.
  - Bulk edit forms are now submitted using Alpine.bind.
- Merges the `.js` includes into one file.
- Save buttons use the `form` attribute instead of JavaScript to save the form.
- Global values are assigned in one file instead of multiple.
- `randomString2()` is now `randomString()`.
- Lots of jQuery code replaced by vanilla JS.

**Review by AlecRust — APPROVED** [review]

**Review by jaapmarcus — APPROVED** [review]

**Comment by ScIT-Raphael** [comment]
@jaapmarcus Please have a look at my resolve for the merge conflict - if all looks well, please squash and merge it.

**Comment by imjuniper** [comment]
Looks good to me!

**Pull request merged**